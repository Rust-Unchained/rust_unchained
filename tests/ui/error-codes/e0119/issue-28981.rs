use std::ops::Deref;

struct Foo;

impl<Foo> Deref for Foo { } //~ ERROR E0119

fn main() {}
