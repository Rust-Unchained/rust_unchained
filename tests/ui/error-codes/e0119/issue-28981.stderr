error[E0119]: conflicting implementations of trait `Deref` for type `memchr::cow::CowBytes<'_>`
  --> $DIR/issue-28981.rs:5:1
   |
LL | impl<Foo> Deref for Foo { }
   | ^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: conflicting implementation in crate `memchr`:
           - impl<'a> Deref for memchr::cow::CowBytes<'a>;

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0119`.
